game.py

```diff
@@ -17,25 +17,47 @@ def snowman(snowman_word):
     """Complete the snowman function
     replace "pass" below with your own code
     It should print 'Congratulations, you win!'
-    If the player wins and, 
+    If the player wins and,
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+
+    while (not is_word_guessed(snowman_word, correct_letter_guess_statuses)
+           and len(wrong_guesses_list) != SNOWMAN_MAX_WRONG_GUESSES):
```

Comment:
Great job splitting this compound conditional across multiple lines! According to the PEP8 style guidelines that we use here at Ada, single lines of code should not be more than 79 characters. You can use this documentation for reference: https://peps.python.org/pep-0008/#maximum-line-length